CTP/shell/src/com/navercorp/cubridqa/shell/common/Constants.java

```diff
@@ -35,14 +35,14 @@
 	
 	public final static String FM_DATE_COREDIR="yyyyMMdd";
 	
-	public static final String DIR_CONF = "./conf";
-	public static final String DIR_LOG_ROOT = "./log";
-	
 	public static final String TC_RESULT_OK = "OK";
 	public static final String TC_RESULT_NOK = "NOK";
 	
 	public static final String RETRY_FLAG = "Retry_Count";
-
+	
+	public static final String RUNTIME_LOG_DIR_HOME = "result/shell";
```

Comment:
May remove both RUNTIME_LOG_DIR_HOME and CURRENT_LOG_DIR.

Replace with fetching from Context.java